utilities/DateUtilities.ts

```diff
@@ -117,17 +121,36 @@ export class DateUtilities {
     return true;
   }
 
-  public static NowToGbDateTimeString() {
-    const timeZoneOptions: any = {
-      timeZone: "Europe/London",
-      hour12: false,
-      hour: "2-digit",
-      minute: "2-digit",
-      second: "2-digit"
-    };
-    const gbDate = new Date().toLocaleDateString("en-GB");
-    const gbTime = new Date().toLocaleTimeString("en-GB", timeZoneOptions);
-    return `${gbDate} ${gbTime}`;
+  private static isAmbiguousOctoberDate(date: day.Dayjs): boolean {
```

Comment:
Ah yes. Don't worry, Single/Double Summer Time (SDST) will definitely not sort this out either :trollface: